shop/shop/src/main/java/ro/msg/learning/shop/model/Order.java

```diff
@@ -0,0 +1,20 @@
+package ro.msg.learning.shop.model;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Set;
+
+@Entity
+@Data
+public class Order extends BaseEntity{
+    @ManyToOne
+    Location shippedfrom;
```

Comment:
Don't forget to make all the entity attributes private. You have `@Data` which is creating the getters and setters for them